Packs/CommonScripts/Scripts/UnEscapeIPv6/UnEscapeIPv6.py

```diff
@@ -0,0 +1,38 @@
+import demistomock as demisto
+
+
+beginning_characters_list = ['(', '[', ' ', '-', '\'', '"', '.', ',', '`']
+ending_characters_list = [')', ']', ' ', '-', '\'', '"', '.', ',', '`']
+
+
+def clearing_the_address_from_unrelated_characters(ipv6_address: str) -> str:
+    """Returns the address without the unrelated characters
+
+    :type ipv6_address: ``str``
+    :param ipv6_address: The address with unrelated characters - (::1234)
+
+    :return: The address without the unrelated characters - ::1234
+    :rtype: ``str``
+    """
+
+    if ipv6_address[0] in beginning_characters_list:
```

Comment:
Is this access safe?
Are we sure that the input will include at least one character?

Reply:
This function will get an IPv6 address with only one character before and/or after it, this is what regex does.